<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sketch2Face</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(213, 236, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
 border: 1px solid black;
border-radius:5px;
background-color: rgb(198, 221, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_image">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 12pt &quot;Segoe UI&quot;;
border-radius:10%;
background-color:#63c132;
color: rgb(255, 255, 255);
background-color: rgb(41, 152, 255);
}
QPushButton:pressed{
	background-color: rgb(0, 114, 187);
}
</string>
    </property>
    <property name="text">
     <string>Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="img1">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>80</y>
      <width>256</width>
      <height>256</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 700 11pt &quot;Segoe UI&quot;;
 border: 1px solid black;
border-radius:5px;
background-color: rgb(198, 221, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pix">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>190</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
font: 700 12pt &quot;Segoe UI&quot;;
border-radius:10%;
background-color:#ff6978;
color: rgb(255, 255, 255);
}
QPushButton:pressed{
	background-color: rgb(227, 93, 107);
}
</string>
    </property>
    <property name="text">
     <string>Pix2Pix</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>731</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(41, 152, 255);
font: 700 italic 15pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Sketch to Face</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>735</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
